boards/arm/colibri_imx7d_m4/colibri_imx7d_m4-pinctrl.dtsi

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2017,2019 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nxp/nxp_imx/mimx7d-pinctrl.dtsi>
+
+&pinctrl {
+	uart2_default: uart2_default {
+		group0 {
+			pinmux = <&mx7d_pad_uart1_rx_data__uart1_dte_tx>,
```

Comment:
I'm guessing uart2?

Reply:
Yup, missed that. Fixed in the latest push.